product-payment.html

```diff
@@ -232,6 +232,84 @@
         </div>
       </div>
     </section>
+    <section class="improve-purchase">
+      <div class="improve-purchase__top">
+        <button class="improve-purchase__buttom">GENARE</button>
```

Comment:
Por lo general los tag en tiendas online se usan como links a la pagina de listado de productos que tienen ese tag
```suggestion
        <a href="#" class="improve-purchase__buttom">GENARE</a>
```